tests/cli.rs

```diff
@@ -112,6 +112,49 @@ fn test_cli_run_with_args() {
   );
 }
 
+#[test]
+fn test_cli_transform() {
+  let arithmetic_program = get_arithmetic_program_path();
+
+  // Test simple program running
+  assert_display_snapshot!(
+    execute_hvmc(&[
+      "transform",
+      "-Opre-reduce",
```

Comment:
you can fix these warnings with an ignore regex like I used for `hvm-lang`: https://github.com/HigherOrderCO/hvm-lang/blob/main/cspell.json#L84